src/components/TasksTable/index.jsx

```diff
@@ -2,30 +2,50 @@ import { Component, Fragment } from 'react';
 import TableCell from '@material-ui/core/TableCell';
 import TableRow from '@material-ui/core/TableRow';
 import Chip from '@material-ui/core/Chip';
+import Drawer from '@material-ui/core/Drawer';
+import IconButton from '@material-ui/core/IconButton';
 import List from '@material-ui/core/List';
 import ListItem from '@material-ui/core/ListItem';
 import ListItemText from '@material-ui/core/ListItemText';
 import Button from '@material-ui/core/Button';
 import { withStyles } from '@material-ui/core/styles';
 import Typography from '@material-ui/core/Typography';
 import Toolbar from '@material-ui/core/Toolbar';
-import IconButton from '@material-ui/core/IconButton';
 import FilterVariantIcon from 'mdi-react/FilterVariantIcon';
 import LinkIcon from 'mdi-react/LinkIcon';
+import InformationVariantIcon from 'mdi-react/InformationVariantIcon';
+import CloseIcon from 'mdi-react/CloseIcon';
 import { withRouter } from 'react-router-dom';
 import { arrayOf, object } from 'prop-types';
 import { camelCase } from 'change-case';
-import { formatDistance } from 'date-fns';
+import { formatDistance, differenceInCalendarDays } from 'date-fns';
+import { memoizeWith, omit, pipe, sort as rSort, map } from 'ramda';
 import MenuItem from '@material-ui/core/MenuItem';
 import TextField from '@material-ui/core/TextField';
-import { memoizeWith, omit, pipe, sort as rSort, map } from 'ramda';
 import { stringify, parse } from 'qs';
 import classNames from 'classnames';
 import DataTable from '../DataTable';
 import sort from '../../utils/sort';
 import { unassigned, assigned } from '../../utils/assignmentFilters';
 import { ASSIGNEE, ALL_PROJECTS } from '../../utils/constants';
 
+const getSuggestions = item => {
+  const daysSinceLastUpdate = differenceInCalendarDays(
+    new Date(),
+    item.lastUpdated
+  );
+
+  if (item.assignee === '-' && daysSinceLastUpdate < 90) {
+    return 'The task is unassigned. Go ahead!';
```

Comment:
The task is assigned to nobody. Are you interested? Ask in the comments to have it assigned to you.